app/src/main/kotlin/com/wire/android/notification/broadcastreceivers/DeclineIncomingCallReceiver.kt

```diff
@@ -56,35 +57,32 @@ class DeclineIncomingCallReceiver : BroadcastReceiver() { // requires zero argum
     @ApplicationScope
     lateinit var coroutineScope: CoroutineScope
 
+    @Inject
+    lateinit var callNotificationManager: CallNotificationManager
+
     override fun onReceive(context: Context, intent: Intent) {
-        val conversationId: String = intent.getStringExtra(EXTRA_CONVERSATION_ID) ?: return
-        appLogger.i("CallNotificationDismissReceiver: onReceive, conversationId: $conversationId")
+        val conversationIdString: String = intent.getStringExtra(EXTRA_CONVERSATION_ID) ?: return
+        appLogger.i("CallNotificationDismissReceiver: onReceive, conversationId: ${conversationIdString.obfuscateId()}")
         coroutineScope.launch(Dispatchers.Default) {
-            val userId: QualifiedID? = intent.getStringExtra(EXTRA_RECEIVER_USER_ID)?.toQualifiedID(qualifiedIdMapper)
-            val sessionScope =
-                if (userId != null) {
-                    coreLogic.getSessionScope(userId)
-                } else {
-                    val currentSession = coreLogic.globalScope { session.currentSession() }
-                    if (currentSession is CurrentSessionResult.Success) {
-                        coreLogic.getSessionScope(currentSession.accountInfo.userId)
-                    } else {
-                        null
+            val userId: UserId? = intent.getStringExtra(EXTRA_RECEIVER_USER_ID)?.toQualifiedID(qualifiedIdMapper)
```

Comment:
What is the situation that receiver id can be null? if not we should somehow handle it, minimally log the error

Reply:
Oh, right, thanks for noticing that! Now it can't, previously both parameters were nullable but it was wrong, so we don't even need to take `userId` from session as it always should be passed